### DISCUSSION TITLE
reportWebVitals has no support for delta value

### DISCUSSION THREAD

**Discussion by dainemawer** [vercel/next.js#44953] [Ideas]
### Verify canary release

- [X] I verified that the issue exists in Next.js canary release

### Provide environment information

Operating System:
      Platform: darwin
      Arch: arm64
      Version: Darwin Kernel Version 21.4.0: Fri Mar 18 00:47:26 PDT 2022; root:xnu-8020.101.4~15/RELEASE_ARM64_T8101
    Binaries:
      Node: 17.8.0
      npm: 8.5.5
      Yarn: N/A
      pnpm: N/A
    Relevant packages:
      next: 12.1.5
      react: 18.0.0
      react-dom: 18.0.0

### What browser are you using? (if relevant)

Chrome 100.0.4896.75

### How are you deploying your application? (if relevant)

Vercel

### Describe the Bug

According to NextJS's documentation around measuring performance (https://nextjs.org/docs/advanced-features/measuring-performance) its indicated that the `reportWebVitals` function leverages `web-vitals.js` under the hood. The `web-vitals.js` offers support for `value` and `delta` however NextJS only exposes the `value` and not `delta` - is there any reason why this was left out? 

Im busy adding some functionality to a site and we need to get the `delta` for certain Web Vitals. 

### Expected Behavior

```
// ⛔  Current Function Definition
export function reportWebVitals({ id, name, label, value }) {}
```

```
// ✅  Expected Function Definition
export function reportWebVitals({ id, name, label, value, delta }) {
```

### To Reproduce

1. Setup a new NextJS project
2. Create a custom `_app.js` 
3. Add the `export function reportWebVitals(metric) {}`
4. No value available for `delta`

**Comment by rmachado-studocu** [comment]
Actually to be more precise, the `web-vitals` package provides the following `Metric` interface:

```ts
export interface Metric {
    name: 'CLS' | 'FCP' | 'FID' | 'LCP' | 'TTFB';
    value: number;
    delta: number;
    id: string;
    entries: (PerformanceEntry | FirstInputPolyfillEntry | NavigationTimingPolyfillEntry)[];
}
```

So, both the `delta` and the `entries` are missing in the `NextWebVitalsMetric` interface.

**Comment by bordeo** [comment]
[Simo Ahava](https://twitter.com/SimoAhava) on it's blog suggests to [use delta for CLS](https://www.simoahava.com/analytics/track-core-web-vitals-in-ga4-with-google-tag-manager/#:~:text=I%E2%80%99m%20setting,than%20the%20deltas.). 
`delta` should be exposed to `reportWebVitals`.

I made a PR to expose `delta` #46908
